department-of-veterans-affairs/va.gov-cms
# VACMS-7926 Remove help text for log in password field

## Description

Closes #7926. Also moves the relevant code for the new login page logic from the `vagovclaro.theme` file into the `va_gov_login.module` where it makes more sense for it to live.

## Testing done
Verified that the help text was gone from the password field & that the login page still renders as expected with the code moving locations.

## Screenshots
Fixed:
<img width="727" alt="Screen Shot 2022-03-03 at 3 21 01 PM" src="https://user-images.githubusercontent.com/11279744/156670253-9393bee9-401d-42b7-9ea0-69ecefa0e350.png">

Before:
<img width="672" alt="Screen Shot 2022-03-03 at 3 23 04 PM" src="https://user-images.githubusercontent.com/11279744/156670256-bee5e7e0-749c-4a8f-a82f-624ae3105033.png">


## QA steps
- Visit the PR environment, click the "developer log in" button in the top right (this is a PR environment, PIV enabled)
- See that the help text for the password field is gone (AC for the ticket)

### Definition of Done

### Select Team for PR review

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
